scripts/convert_wav_to_g711wav.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2020 NVIDIA Corporation
```

Comment:
Can you add the entire copyright?

Reply:
Done! Looks like I copy pasted from an older file.